src/main/scala/circt/Convention.scala

```diff
@@ -0,0 +1,52 @@
+package chisel3.circt
```

Comment:
I think this is the first time we have a `package chisel3.circt`, everything else is just `package circt`?

I'm fine with a `chisel3.annotations` or `circt.annotations` or just `circt`. If we decide to put it in a subpackage we should (proper scala style) mirror that in the directory structure.